shared/prometheus/service.go

```diff
@@ -97,7 +97,7 @@ func (s *Service) healthzHandler(w http.ResponseWriter, r *http.Request) {
 			if s.Status {
 				status = "OK"
 			} else {
-				status = "ERROR " + s.Err
+				status = "ERROR, " + s.Err
```

Comment:
what do you think about using a semi-colon instead ?
```suggestion
				status = "ERROR: " + s.Err
```

Reply:
I dont have a hard preference and applied the changes. Keep in mind that now it has 2 semi colons:

`*prometheus.mockService: ERROR: something is wrong`

Instead of:

`*prometheus.mockService: ERROR, something is wrong`

Still ok with you?

Reply:
Ah good point , the double colons is not very nice here. Although `ERROR,` with the comma still doesn't feel like the correct way. I have no hard preference now, a comma is fine too lets just keep it as it is here.